react-charts/bar-chart/step7.md

```diff
@@ -1,56 +1,18 @@
-Now we'll add a legend to the bar chart...
+Change the area chart theme color.
 
 ## Task
 
-1) Make sure the App.js file is still open
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-2) Locate the code that looks like the following:
+2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>
 
-<pre class="file">
-&lt;Chart
-  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
-  domainPadding={{ x: [30, 25] }}
-  padding={{
-    bottom: 75,
-    left: 75,
-    right: 50,
-    top: 50
-  }}
-  height={250}
-  width={600}
-&gt;
-</pre>
-
-3) Replace the following property in that section:
+The `themeColor` property specifies the theme color. Valid values are
+'blue,' 'green,' 'multi,' etc. It should look like the following:
 
 <pre class="file" data-target="clipboard">
-padding={{
-  bottom: 75,
-  left: 75,
-  right: 200,
-  top: 50
-}}
+themeColor={ChartThemeColor.gold}

```

Comment:
Why was it necessary to change this?

Reply:
It was @christiemolloy idea. I agree that it's can be better to have different themes in the different tutorials.

Reply:
I don't believe we need to pick different colors for each tutorial. Unless you're following guidelines, there may be recommendations for chart types.